Sources/Action/UIKitExtensions/UIBarButtonItem+Action.swift

```diff
@@ -37,5 +38,28 @@ public extension Reactive where Base: UIBarButtonItem {
             }
         }
     }
+    public func bindToAction <Input,Output>(_ action:Action<Input,Output>?,_ inputTransform: @escaping (Base) -> (Input))   {
```

Comment:
@ashfurrow seems legit, but I just want to point out that `CocoaAction`s are optional for buttons (and I think it's needed, since they are defined through variables ad a `UIButton` doesn't have any action assinged when it's created).
Should we create sometning like `func unbindAction()` that calls `bindTo` with an empty Action? 

https://github.com/RxSwiftCommunity/Action/blob/e606d7db0c891fc9f242f49a8bf646ba793508bc/Sources/Action/UIKitExtensions/UIButton%2BRx.swift#L11